### PR TITLE
feat: reorder exhibitors

### PR DESCRIPTION
**Pull request by maze-runnar**
Fixes https://github.com/fossasia/open-event-frontend/issues/6681
Screenshot from 2021-02-15 21-02-04
Screenshot from 2021-02-15 21-01-53
Screenshot from 2021-02-15 21-01-49
Screenshot from 2021-02-15 21-01-39
Screenshot from 2021-02-16 11-42-26

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The unit tests pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

- [ ] All the functions created/modified in this PR contain relevant docstrings.

### CODE REVIEW

File: app/api/custom/events.py [hunk 1]
```diff
@@ -128,3 +130,47 @@ def reorder_speakers(event_id):
     db.session.commit()
 
     return jsonify({'success': True, 'updates': updates})
+
+
+@events_routes.route('/<string:event_identifier>/reorder-exhibitors', methods=['POST'])
+@to_event_id
+def reorder_exhibitors(event_id):
```
Comment (iamareebjamal) [inline app/api/custom/events.py:137]: Only coorganizer should be able to do this

Comment (maze-runnar) [reply app/api/custom/events.py:137]: it giving error I was talking about.

Comment (maze-runnar) [reply app/api/custom/events.py:137]: AssertionError: View function mapping is overwriting an existing endpoint function: events_routes.fwrapper

File: app/api/schema/exhibitors.py [hunk 2]
```diff
@@ -36,6 +37,7 @@ class Meta:
     slides_url = fields.Url(allow_none=True)
     contact_email = fields.Str(allow_none=True)
     contact_link = fields.Str(allow_none=True)
+    order = fields.Integer(allow_none=True, default=0)
```
Comment (iamareebjamal) [inline app/api/schema/exhibitors.py:40]: Already `position` column is present

File: app/models/exhibitor.py [hunk 3]
```diff
@@ -23,6 +23,7 @@ class Status:
     description = db.Column(db.String)
     url = db.Column(db.String)
     position = db.Column(db.Integer, nullable=False, default=0, server_default='0')
+    order = db.Column(db.Integer, default=0, nullable=False)
```
Comment (iamareebjamal) [inline app/models/exhibitor.py:26]: Use position column

File: app/api/schema/exhibitors.py (hunk 2 above)
Comment (maze-runnar) [reply app/api/schema/exhibitors.py:40]: I was confused with position and order.

File: migrations/versions/rev-2021-02-15-17:29:51-ff6fa5834f7f_.py [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+
+# revision identifiers, used by Alembic.
+revision = 'ff6fa5834f7f'
+down_revision = '6f68fb58f228'
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('exhibitors', sa.Column('order', sa.Integer(), nullable=False, server_default='0'))
```
Comment (iamareebjamal) [inline migrations/versions/rev-2021-02-15-17:29:51-ff6fa5834f7f_.py:21]: Use position column

File: app/api/custom/events.py (hunk 1 above)
Comment (iamareebjamal) [reply app/api/custom/events.py:137]: Check the position of decorator. Match with speaker reorder function

Comment (maze-runnar) [reply app/api/custom/events.py:137]: I added it in same way as speaker reorder then it was giving errror and the I removed that, and error was gone.

Comment (maze-runnar) [reply app/api/custom/events.py:137]: giving error -
```
@to_event_id
@is_coorganizer
def reorder_exhibitors(event_id):
```

not giving error
```
@to_event_id
def reorder_exhibitors(event_id):
```

**Comment by iamareebjamal** [comment]
Check now

**Comment by maze-runnar** [comment]
> Check now

worked

**Pull request merged**